decidim-core/app/views/decidim/shared/_orders.html.erb

```diff
@@ -1,9 +1,10 @@
 <div class="row collapse order-by">
   <div class="order-by__dropdown">
     <span class="order-by__text"><%= t("#{i18n_scope}.label") %></span>
-    <ul class="dropdown menu" data-dropdown-menu data-close-on-click-inside="false">
-      <li>
-        <%= order_link order, i18n_scope: i18n_scope %>
+    <ul class="dropdown menu" data-dropdown-menu>
```

Comment:
We needed the `data-close-on-click-inside="false"` to fix some weird bug with the dropdowns: https://github.com/decidim/decidim/pull/977. Is this solved already? We might have updated foundation and forgot to remove this.